extras/AFC.py

```diff
@@ -813,7 +813,8 @@ def get_status(self, eventtime):
             str["system"]["extruders"][EXTRUDE]['tool_end_sensor']   = True == CUR_EXTRUDER.tool_end_state   if CUR_EXTRUDER.tool_end   is not None else False
             if CUR_EXTRUDER.buffer_name != None:
                 str["system"]["extruders"][EXTRUDE]['buffer']   = CUR_EXTRUDER.buffer_name
-                str["system"]["extruders"][EXTRUDE]['buffer_status']   = CUR_EXTRUDER.buffer.buffer_status()
+                CUR_BUFFER = self.printer.lookup_object('AFC_buffer ' + CUR_EXTRUDER.buffer_name)
```

Comment:
You should not need to change these 2 lines as `CUR_EXTRUDER` already holds the object for buffer

Reply:
Main has these same changes as well

Reply:
ignore this comment for now